src/main/java/com/indiealexh/MainVerticle.java

```diff
@@ -0,0 +1,243 @@
+package com.indiealexh;
+
+import com.indiealexh.models.FireData;
+import com.indiealexh.models.GeoLocation;
+import com.indiealexh.models.StateFireData;
+import io.vertx.core.AbstractVerticle;
+import io.vertx.core.CompositeFuture;
+import io.vertx.core.Future;
+import io.vertx.core.buffer.Buffer;
+import io.vertx.core.http.HttpClient;
+import io.vertx.core.http.HttpHeaders;
+import io.vertx.core.http.HttpServer;
+import io.vertx.core.json.JsonObject;
+import io.vertx.core.logging.Logger;
+import io.vertx.core.logging.LoggerFactory;
+import io.vertx.ext.web.Router;
+import io.vertx.ext.web.RoutingContext;
+import io.vertx.ext.web.client.HttpResponse;
+import io.vertx.ext.web.client.WebClient;
+import org.w3c.dom.Document;
+import org.w3c.dom.Node;
+import org.w3c.dom.NodeList;
+
+import javax.xml.parsers.DocumentBuilder;
+import javax.xml.parsers.DocumentBuilderFactory;
+import java.io.ByteArrayInputStream;
+import java.io.InputStream;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class MainVerticle extends AbstractVerticle {
```

Comment:
It's a matter of taste, but it's common in Java to place the opening bracket on the next line. Especially since you've added a extra break to improve readability, I'd recommend it.

Reply:
I use the Google Style guide for Java.